satellite/console/consoleweb/server.go

```diff
@@ -192,33 +192,32 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 
 	var projectID *uuid.UUID
 	var since, before time.Time
+	var bucketRollups []console.BucketUsageRollup
 
-	tokenCookie, err := r.Cookie("tokenKey")
+	defer func() {
+		if err != nil {
+			// TODO: use http.StatusUnauthorized status when appropriate page will be created
+			server.serveError(w, r, http.StatusNotFound)
+		}
+	}()
+
+	host, _, err := net.SplitHostPort(r.Host)
 	if err != nil {
 		server.log.Error("bucket usage report error", zap.Error(err))
+		return
```

Comment:
Why this error will turn in to a `StatusUnauthorized`?

Reply:
?

Reply:
This handled by the `defer` which currently is `NotFound` but it will be changed by `Unauthorized` when the page is created.

Isn't this an `InternalServerError`.

Although all who create the page and do refactoring may have to go and check each return I think that it's quite easy to forget to do so and only update the `defer` states which is where the `TODO` comment is placed.